src/Tests/CollectionRuleActions.UnitTests/Egress/Pipe/PipeEgressOptions.cs

```diff
@@ -4,7 +4,7 @@
 
 using System.IO.Pipelines;
 
-namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests.Egress.Pipe
+namespace CollectionRuleActions.UnitTests
```

Comment:
Just wanted to double check that trimming off the namespace on a few of these files was intentional, feel free to ignore.

Reply:
Yes, intentional, to match the name of the assembly to which it was moved.